prisma/migrations/20230221114026_drop_not_null_constraint_website_type_description/migration.sql

```diff
@@ -0,0 +1,5 @@
+SELECT template.run_migration('
+-- AlterTable
+ALTER TABLE "recruitments" ALTER COLUMN "website" DROP NOT NULL,
+ALTER COLUMN "description" DROP NOT NULL;
```

Comment:
Coi có thiếu cái drop not null của cái type ko đó

Reply:
Prisma bug nha 😀